hw/top_earlgrey/data/chip_conn_testplan.hjson

```diff
@@ -23,6 +23,98 @@
       tags: ["conn"]
     }
 
+    /////////////////////////
+    // ast_infra.csv       //
+    /////////////////////////
+    {
```

Comment:
there's a todo item all the way on the bottom for the `fips` connection, can you remove that as part of this PR?

Reply:
I saw that one. it looks like that item needs to be updated. there is a `.csv` setup file for `entropy_src` connectivity tests that has not been referenced in the testplan. I will add this as a separate commit to this PR.

Reply:
see previous commit in this PR.